app/models/data_export.rb

```diff
@@ -86,7 +86,7 @@ class DataExport < ApplicationRecord
       class: SupportInterface::ProviderAccessControlsExport,
     },
     locations_export: {
-      name: 'Locations export',
+      name: 'Locations',
```

Comment:
The word export is added later, so this was rendering as "Locations export export".